users/login.php

```diff
@@ -0,0 +1,73 @@
+<?php
+    session_start();
+    if(isset($_SESSION['user_id']) != "") {
+        header("Location: user_profile.php");
+    }
+  $title = 'Login';
+  include('header.php');
+?>
+<div class="container">
+    <div class="row">
+        <h2 class="pagetitle">Login</h2>
+    </div>
+    <div class="row d-flex">
+	    <div class="flex-50">
+        <?php
+            // Display success or error messages if they are set in the session
+            if (isset($_SESSION['username'])) {    
+            echo $_SESSION['username'];
+        }
+                if (isset($_SESSION['success_message'])) {
+                    echo '<p style="color: green;">' . $_SESSION['success_message'] . '</p>';
+                    unset($_SESSION['success_message']); // Clear the success message after displaying
+                }
+                if (isset($_SESSION['error_message'])) {
+                    echo '<p style="color: red;">' . $_SESSION['error_message'] . '</p>';
+                    unset($_SESSION['error_message']); // Clear the error message after displaying
+                }
+            ?>
+            <form action="login_process.php" method="post">
+                <div class="form-group-row">        
+                    <div class="form-group">
+                        <input type="text" name="username" id="username" placeholder="Username"> 
+                        <p class="text-center">OR</p>
+                        <input type="email" name="email" id="email" placeholder="Email" pattern="/^(?=.*[A-Z])(?=.*\d)(?=.*[!@#$%()]).{6,}$/">

```

Comment:
can we add individual error messages to the fields respectively and clear the error on key up, also please higight the textbox with error border

Reply:
This is done